Dockerfile

```diff
@@ -34,10 +34,7 @@ ENV PATH=$PATH:$JAVA_HOME/bin
 
 # NodeJS
 RUN curl -sL https://deb.nodesource.com/setup_lts.x | bash - \
-    && apt -y install nodejs \
-    && apt -y install ffmpeg \
-    && apt -y install make \
-    && apt -y install build-essential 
```

Comment:
Having these on separate lines can be helpful for readability.

Reply:
This wont be approved at this time. Dockerfiles is something end users don't get to see so it doesnt need to be more readable. And having them as seperate commands can also increase build time and install time.